pages/index.js

```diff
@@ -19,6 +20,7 @@ export default function HomePage() {
         </Modal>
       )}
       <AccountBalance transactions={data} />
+      <TransactionsHeader hasAddButton />
```

Comment:
Now you apply the same concept as before - only in a different component. 
Since you want to see it on the homepage you should render it there. 
(just a suggestion to keep the code clean and to seperate concerns)